listenstore/listenstore/utils.py

```diff
@@ -3,23 +3,32 @@
 import argparse
 from setproctitle import setproctitle
 import json
-
-
-## TODO read config from whereever
-def config():
-    return {
-        "kafka_server": "localhost:9092",
-        "cassandra_server": "localhost",
-        "cassandra_keyspace": "listenbrainz"
-    }
+import ConfigParser
+import os
+
+def config(opt_vars):
+    """ Parse config file else return default values """
```

Comment:
That is again a problem.
We access Listenstore from 2 places, once while storing DB which is from the LB app, hence we can take into account what you said, but when accessing Listenstore for showing listens, we are not accessing it from the flask app. 

This was the problem i faced, and the workaround i found was to keep a copy of configuration in `listenstore.conf` and `config.py` (of the main LB app)